news/l_progress.rst

```diff
@@ -1,4 +1,5 @@
-**Added:** added a helper to list progress by person
+**Added:**
+* a helper that lists progress by person from their projecta
```

Comment:
I think this may need a space before the asterisk.  It is tricky rst formatting where indentation carries meaning.

Reply:
Done.